index.js

```diff
@@ -45,21 +45,8 @@ function convertToSeconds(milliseconds) {
   return Math.floor(milliseconds / 1000)
 }
 
-function getAlertTypeDisplayName(alertType) {
```

Comment:
Made more sense to me to have this in `brave-alert-lib` since that's where the `ALERT_TYPE` type is defined.